src/Controls/src/Core/BoxView/BoxView.cs

```diff
@@ -56,9 +56,13 @@ protected override void OnPropertyChanged([CallerMemberName] string? propertyNam
 
 			if (propertyName == BackgroundColorProperty.PropertyName ||
 				propertyName == ColorProperty.PropertyName ||
+				propertyName == IsVisibleProperty.PropertyName ||
 				propertyName == BackgroundProperty.PropertyName ||
 				propertyName == CornerRadiusProperty.PropertyName)
 				Handler?.UpdateValue(nameof(IShapeView.Shape));
+
+			if(propertyName == IsVisibleProperty.PropertyName)
```

Comment:
Is this necessary?

Reply:
Strangely enough, it is only necessary for Android. For iOS everything works with only updating `Handler?.UpdateValue(nameof(IShapeView.Shape));`  when the visibility changes